packages/babel-parser/src/plugins/flow.js

```diff
@@ -22,12 +22,15 @@ import {
   SCOPE_OTHER,
 } from "../util/scopeflags";
 
-const reservedTypes = [
+const reservedTypes = new Set([
+  "_",
```

Comment:
I wish there was something like https://eslint.org/docs/rules/sort-keys for arrays